raster/identify-raster-cell/build.gradle

```diff
@@ -0,0 +1,77 @@
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.5'
+}
+
+group = 'com.esri.samples'
+
+ext {
+    arcgisVersion = '100.8.0'
+}
+
+javafx {
+    version = "11.0.1"
+    modules = [ 'javafx.controls', 'javafx.fxml' ]
```

Comment:
to be sure you should add all modules.  If anyone adds functionality to the app it will break.  We should add all modules to all samples by default.

```
    modules = [ 'javafx.controls',
                'javafx.base',
                'javafx.graphics',
                'javafx.fxml',
                'javafx.web',
                'javafx.media']

```